beast/plotting/plot_region_diags.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
+"""
+Show diagnostic plots for a square region in two parameters
+"""
+
+from __future__ import print_function
+
+import matplotlib.pyplot as plt
+from astropy.table import Table
+from matplotlib.colors import LogNorm
+from matplotlib import rc
+import numpy as np
+
+# local imports
+from beastplotlib import (fancify_colname, initialize_parser,
+                          plot_generic, set_params)
+
+def make_region_diag_plots(stats, xparam='logT', yparam='logL',
+                           suffix='Exp', xrange=[3.8,4.0], yrange=[1.0,6.0],
```

Comment:
``xrange`` is a builtin function in Python 2; it might be good to rename this and ``yrange`` to avoid confusion.